Rise-Vision/common-header
# Add "settings" to company writable fields

## Description
Include the `settings` field to company writable fields.

## Motivation and Context
We are going to store Branding information within the `settings` map that is part of the Company object.

## How Has This Been Tested?
This has been tested on stage-1 via the prototype branch for Branding. Support for Branding fields via `company.settings` has already been released in the Core.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
**Manual Test**  As per the above
**Automated Test** No relevant tests are needed for this
**Monitoring** This code will not be used unless that field is saved within the UI
**Rollback** Rollback can be done by reverting the PR
**Documentation** Documentation for the Epic: https://docs.google.com/document/d/1XZUcQ6D9oWjMxrllbZKZLCDV3D8W89OyqrUMwVjEp_8/edit#
**Internal Communication** Will notify interested parties once deployed

- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a
